cpu/atmega_common/include/periph_cpu_common.h

```diff
@@ -33,6 +33,22 @@ extern "C" {
 #define GPIO_PIN(x, y)          ((x << 4) | y)
 
 /**
+ * @brief   Override the GPIO flanks
+ *
+ * This device has an additional mode in which the interrupt is triggered
+ * when the pin is low. 
+ * @{
+ */
+#define HAVE_GPIO_FLANK_T
+typedef enum {
+    GPIO_LOW = 0,           /**< emit interrupt when pin low */
+    GPIO_BOTH = 1,          /**< emit interrupt on both flanks */
+    GPIO_FALLING = 2,       /**< emit interrupt on falling flank */
+    GPIO_RISING = 3,        /**< emit interrupt on rising flank */
```

Comment:
minor nit (and personal preference by me): either omit explicit definition of values *or* align them - looks nicer 😉

Reply:
Good point, will change :+1: